lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -116,6 +120,7 @@ class OffscreenImages extends ByteEfficiencyAudit {
     // TODO(phulce): move this to always use lantern
     const settings = context.settings;
     return artifacts.requestFirstCPUIdle({trace, devtoolsLog, settings}).then(firstInteractive => {
+      // @ts-ignore - see above TODO.
```

Comment:
right now this is actually an `LH.Artifacts.Metric`, so `firstInteractive.timestamp` does exist. When it switches to lantern (see above), it's going to have to switch off of `timestamp` anyways, so didn't seem worth fixing